public/css/print.css

```diff
@@ -1,47 +1,49 @@
+@font-face {
```

Comment:
Do we not sass in this project?

Reply:
Nope. KISS principle. Same reason we don't Vue.JS it instead of trusty old jquery :-)

Reply:
hmmm ok. :)